contracts/interfaces/events/IBosonAccountEvents.sol

```diff
@@ -22,6 +22,11 @@ interface IBosonAccountEvents {
         BosonTypes.AuthToken authToken,
         address indexed executedBy
     );
+    event SellerUpdateAddressesRequested(
+        uint256 indexed sellerId,
+        BosonTypes.SellerUpdate seller,
+        address indexed executedBy
```

Comment:
This is a note to myself, it seems like this could be performed by the authToken too, i guess that should be a part of this struct ...

Reply:
@hswopeams I remove the `UpdateSeller` struct, and now I'm using the `Seller` struct itself. 

I don't think its necessary the pending feature for the AuthToken update